lib/rubocop/extension/generator/generator.rb

```diff
@@ -145,8 +145,10 @@ def rules(_context)
             gem 'rspec'
           RUBY
 
-          if Gem::Version.new(Bundler::VERSION) >= Gem::Version.new('2.3.9')
-            patch 'README.md', /\$ bundle add (.*)$/, '$ bundle add \1 --require=false'
+          # Bundler's generated README.md has had two styles of gem declarations.
+          # See: https://github.com/rubygems/rubygems/commit/c805e9b558
+          if Gem::Version.new(Bundler::VERSION) >= Gem::Version.new('2.6.0')
```

Comment:
Shouldn't version 2.3.9 be kept for backward compatibility?

```suggestion
          if Gem::Version.new(Bundler::VERSION) >= Gem::Version.new('2.3.9')
```

Reply:
🤦 yes of course. I forgot in my spike I overwrote the existing condition. Added this as a a third branch and repushed. Sorry!